tests/rkt-monitor/build-stresser.sh

```diff
@@ -6,30 +6,36 @@ if ! [[ "$0" =~ "tests/rkt-monitor/build-stresser.sh" ]]; then
 	exit 255
 fi
 
-stressers=(cpu mem log)
+stressers="cpu mem log"
 
 if [ -z "${1}" ]; then
-    echo Specify one of \""${stressers[@]}"\"
+    echo Specify one of \""${stressers[@]}"\" or all
     exit 1
 fi
 
-
 echo "Building worker..."
 make rkt-monitor
 
 acbuildEnd() {
-    export EXIT=$?
-    acbuild --debug end && exit $EXIT
+    if [ -d ".acbuild" ]; then
+      acbuild --debug end
+    fi
 }
 
-acbuild --debug begin
-
-trap acbuildEnd EXIT
-
-acbuild --debug set-name appc.io/rkt-"${1}"-stresser
-
-acbuild --debug copy build-rkt-1.13.0+git/target/bin/"${1}"-stresser /worker
-
-acbuild --debug set-exec -- /worker
+buildImages() {
+    acbuild --debug begin
+    acbuild --debug set-name appc.io/rkt-"${1}"-stresser
+    acbuild --debug copy build-rkt-1.13.0+git/target/bin/"${1}"-stresser /worker
+    acbuild --debug set-exec -- /worker
+    acbuild --debug write --overwrite "${1}"-stresser.aci
+    acbuildEnd
```

Comment:
The `set -e` makes the script exit when an acbuild command fails, and the `trap` makes the script run `acbuild end` when the script is about to exit. This means in the event of an error acbuild is able to clean up after itself.

Removing the trap will prevent the script from performing cleanup on error, so I'd highly recommend putting that back.